FirebaseFunctionsSwift/Sources/HTTPSCallable.swift

```diff
@@ -93,6 +94,24 @@ public class HTTPSCallable: NSObject {
     }
   }
 
+  /**
+   * Executes this Callable HTTPS trigger asynchronously.
+   *
+   * The request to the Cloud Functions backend made by this method automatically includes a
+   * Firebase Installations ID token to identify the app instance. If a user is logged in with
+   * Firebase Auth, an auth ID token for the user is also automatically included.
+   *
+   * Firebase Cloud Messaging sends data to the Firebase backend periodically to collect information
+   * regarding the app instance. To stop this, see `Messaging.deleteData()`. It
+   * resumes with a new FCM Token the next time you call this method.
+   *
+   * @param completion The block to call when the HTTPS request has completed.
+   */
+  @objc(callWithCompletion:) public func __call(completion: @escaping (HTTPSCallableResult?,
```

Comment:
ObjC only API

Reply:
Nit: please add a comment in the doc itself mentioning it should only be used from ObjC